charts/victoria-metrics-auth/values.yaml

```diff
@@ -0,0 +1,194 @@
+  enabled: false
+  # minAvailable: 1
+  # maxUnavailable: 1
+  labels: {}
+
+# -- WARN: need to specify at least one remote write url
+remoteWriteUrls: []
+# remoteWriteUrls:
+#   - http://vm-insert:8480/insert/0/prometheus
+#   - http://prometheus:8480/insert/0/prometheus
+
+extraArgs:
+  envflag.enable: "true"
+  envflag.prefix: VM_
+  loggerFormat: json
+  # promscrape.maxScrapeSize: "167772160"
+
+# -- Additional environment variables (ex.: secret tokens) https://github.com/VictoriaMetrics/VictoriaMetrics#environment-variables
+env: []
+
+# -- Additional hostPath mounts
+extraHostPathMounts: []
+  # - name: certs-dir
+  #   mountPath: /etc/kubernetes/certs
+  #   subPath: ""
+  #   hostPath: /etc/kubernetes/certs
+  #   readOnly: true
+
+# -- Extra Volumes for the pod
+extraVolumes: []
+  # - name: example
+  #   configMap:
+  #     name: example
+
+# -- Extra Volume Mounts for the container
+extraVolumeMounts: []
+  # - name: example
+  #   mountPath: /example
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+service:
+  enabled: true
+  annotations: {}
+  extraLabels: {}
+  clusterIP: ""
+  ## Ref: https://kubernetes.io/docs/user-guide/services/#external-ips
+  ##
+  externalIPs: []
+  loadBalancerIP: ""
+  loadBalancerSourceRanges: []
+  servicePort: 8427
+  type: ClusterIP
+ingress:
+  enabled: false
+  annotations: {}
+  #   kubernetes.io/ingress.class: nginx
+  #   kubernetes.io/tls-acme: 'true'
+
+  extraLabels: {}
+  hosts: []
+  #   - name: vmauth.local
+  #     path: /
+  #     port: http
+  tls: []
+  #   - secretName: vmauth-ingress-tls
+  #     hosts:
+  #       - vmauth.local
+
+# -- We usually recommend not to specify default resources and to leave this as a conscious choice for the user. This also increases chances charts run on environments with little resources, such as Minikube. If you do want to specify resources, uncomment the following lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+resources: {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+# -- Annotations to be added to the deployment
+annotations: {}
+
+# -- Annotations to be added to pod
+podAnnotations: {}
+
+# -- NodeSelector configurations. Ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+# -- Tolerations configurations. Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+tolerations: []
+  # - key: "key"
+  #   operator: "Equal|Exists"
+  #   value: "value"
+  #   effect: "NoSchedule|PreferNoSchedule"
+
+# -- Affinity configurations
+affinity: {}
+
+serviceMonitor:
+  enabled: false
+  extraLabels: {}
+  annotations: {}
+#    interval: 15s
+#    scrapeTimeout: 5s
+
+persistence:
```

Comment:
i guess, pvc not needed for vmauth

Reply:
Thanks @f41gh7, I will remove this in the next commit.